reaktive/src/commonMain/kotlin/com/badoo/reaktive/scheduler/Schedulers.kt

```diff
@@ -59,22 +78,38 @@ expect fun createComputationScheduler(): Scheduler
  */
 expect fun createTrampolineScheduler(): Scheduler
 
+/**
+ * Creates a new instance of Trampoline [Scheduler]
+ */
+expect fun createSingleScheduler(): Scheduler
+
+/**
+ * Creates a new instance of New Thread [Scheduler]
+ */
+expect fun createNewThreadScheduler(): Scheduler
+
 /**
  * Overrides [Scheduler]s if they were not created yet
  *
  * @param main a factory for Main [Scheduler], if not set then default factory will be used
  * @param computation a factory for Computation [Scheduler], if not set then default factory will be used
  * @param io a factory for IO [Scheduler], if not set then default factory will be used
  * @param trampoline a factory for Trampoline [Scheduler], if not set then default factory will be used
+ * @param single a factory for Single [Scheduler], if not set then default factory will be used
+ * @param newThread a factory for New Thread [Scheduler], if not set then default factory will be used
  */
 fun overrideSchedulers(
     main: () -> Scheduler = ::createMainScheduler,
     computation: () -> Scheduler = ::createComputationScheduler,
     io: () -> Scheduler = ::createIoScheduler,
-    trampoline: () -> Scheduler = ::createTrampolineScheduler
+    trampoline: () -> Scheduler = ::createTrampolineScheduler,
+    single: () -> Scheduler = ::createSingleScheduler,
```

Comment:
These changes are kinda braking, not sure whether it's fine or not. Perhaps it's fine.

Reply:
I don't think it is critical. Just don't forget to mention it in changelog.